source/common/runtime/runtime_features.cc

```diff
@@ -82,6 +84,8 @@ FALSE_RUNTIME_GUARD(envoy_reloadable_features_thrift_connection_draining);
 // TODO(birenroy) flip after a burn-in period
 // Requires envoy_reloadable_features_http2_new_codec_wrapper to be enabled.
 FALSE_RUNTIME_GUARD(envoy_reloadable_features_http2_use_oghttp2);
+// Used to track if runtime is initialized.
+FALSE_RUNTIME_GUARD(envoy_reloadable_features_runtime_initialized);
```

Comment:
nit: it seems like it would be better to have this be some unrelated static since technically this could I guess be overwritten by an actual configuration, but not a big deal. Up to you if you want to change it or not.

Reply:
yeah it can get overriden by config but it can't be until after runtime is initialized at which point you might as well just use the actual handle to set size high enough.
intentionally used a runtime guard so it gets reset between tests